test/service/src/genesis.rs

```diff
@@ -14,10 +14,10 @@
 // You should have received a copy of the GNU General Public License
 // along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
 
-use codec::Encode;
 use cumulus_primitives::{genesis::generate_genesis_block, ParaId};
 use cumulus_test_runtime::Block;
 use polkadot_primitives::v0::HeadData;
+use sp_core::Encode;
```

Comment:
What? Why you start importing this from `sp_core`?

Reply:
It doesn't work with the one of codec for some reason... I will check the dependencies

Reply:
```
error[E0599]: no method named `encode` found for reference `&sp_runtime::generic::Header<u32, BlakeTwo256>` in the current scope
  --> test/service/src/genesis.rs:27:37
   |
27 |     let genesis_state = block.header().encode();
   |                                        ^^^^^^ method not found in `&sp_runtime::generic::Header<u32, BlakeTwo256>`
   | 
  ::: /home/cecile/.cargo/git/checkouts/substrate-7e08433d4c370a21/93b231e/primitives/runtime/src/generic/header.rs:40:1
   |
40 | pub struct Header<Number: Copy + Into<U256> + TryFrom<U256>, Hash: HashT> {
   | -------------------------------------------------------------------------
   | |
   | doesn't satisfy `<_ as Deref>::Target = _`
   | doesn't satisfy `_: parity_scale_codec::Encode`
   | doesn't satisfy `_: parity_scale_codec::WrapperTypeEncode`
   |
   = note: the method `encode` exists but the following trait bounds were not satisfied:
           `<sp_runtime::generic::Header<u32, BlakeTwo256> as Deref>::Target = _`
           which is required by `sp_runtime::generic::Header<u32, BlakeTwo256>: parity_scale_codec::Encode`
           `sp_runtime::generic::Header<u32, BlakeTwo256>: parity_scale_codec::WrapperTypeEncode`
           which is required by `sp_runtime::generic::Header<u32, BlakeTwo256>: parity_scale_codec::Encode`
   = help: items from traits can only be used if the trait is in scope
   = note: the following trait is implemented but not in scope; perhaps add a `use` for it:
           `use sp_core::Encode;`
```

Reply:
Ok I found it! It's hmm... single quote... so I failed my regex and one parity codec was not updated XD